packages/lightcone_v2.js/src/sign/exchange.ts

```diff
@@ -25,6 +25,8 @@ import {
   SimpleOrderCancellationReq
 } from "../grpc/proto_gen/service_dex_pb";
 
+const MOCK_EXCHANGE_GAS_LIMIT = "0xf0f09f0";
```

Comment:
We are using a relative large value. Unless, there is a better value. @iwbrhwfy @kongliangzhong